tests/system/src/test/java/test/robot/helloworld/CustomSecurityManagerTest.java

```diff
@@ -201,11 +201,11 @@ private void doTestFullScreenCommon(SecurityManager sm,
                     // avoid the top area as it might contain OS-specific UI (Macs with a notch)
                     y = h / 3;
                 } else {
-                    y = h - 2;
+                    y = h - 5;
                 }
 
                 for (int col = 0; col < 2; col++) {
-                    int x = col == 0 ? 1 : screenWidth.get() - 2;
+                    int x = col == 0 ? 1 : screenWidth.get() - 5;
```

Comment:
For `col == 0` you might want to change the value from `1` to `4` for the same reason you changed the second value.

Reply:
Taking my previous comment to the other query into account, I think this should be more like:

```
                    int x = col == 0 ? screenBound.minX + 4 : screenBounds.maxX - 5;
```

where `screenBounds` is the visual bounds of the screen.